nautobot/dcim/templates/dcim/location.html

```diff
@@ -128,6 +128,40 @@
                 <td>{{ object.contact_email|hyperlinked_email }}</td>
             </tr>
         </table>
+        {% with request.path|add:"?tab=contacts"|urlencode as return_url %}
+            <div class="panel-footer text-right noprint">
+                {% url 'extras:map_contact_from_location' as map_url %}
+                {% with object.pk|urlencode as pk %}
+                {% with object.contact_name|urlencode as name %}
+                {% with object.contact_phone|urlencode as phone %}
+                {% with object.contact_email|urlencode as email %}
+                {% with content_type.id|urlencode as contenttypeid %}
+                {% if object.shipping_address %}
+                    {% with object.shipping_address|urlencode as address %}
+                        {% with map_url|add:"?return_url"|add:return_url|add:"&associated_object_id="|add:pk|add:"&associated_object_type="|add:contenttypeid|add:"&location="|add:pk|add:"&name="|add:name|add:"&phone="|add:phone|add:"&email="|add:email|add:"&address="|add:address as new_map_url %}
```

Comment:
We decided in the management command *not* to auto-convert the shipping address/physical address of a Location to a contact address, right? Should be consistent here as well.